test/Interop/Cxx/reference/Inputs/reference.cpp

```diff
@@ -16,3 +16,19 @@ void setConstStaticIntRvalueRef(const int &&i) { staticInt = i; }
 
 auto getFuncRef() -> int (&)() { return getStaticInt; }
 auto getFuncRvalueRef() -> int (&&)() { return getStaticInt; }
+
+void takeConstPODStructRef(const PODStruct &value) {
```

Comment:
Nit: these can probably all go in the header (makes it easier to find in the future). The source file is just for the statics.